ably-iosTests/ARTTestUtil.m

```diff
@@ -280,4 +281,11 @@ + (void)testRealtimeV2:(XCTestCase *)testCase withDebug:(BOOL)debug callback:(AR
     [testCase waitForExpectationsWithTimeout:[ARTTestUtil timeout] handler:nil];
 }
 
++ (ARTProtocolMessage *)simulateError {
```

Comment:
Should this not be named something `newErrorProtocolMessage` as it's not actually simulating anything until it's used, it's just creating an object